Xamarin.Forms.Platform.UAP/VisualElementRenderer.cs

```diff
@@ -155,7 +155,7 @@ public void SetElement(VisualElement element)
 
 			OnElementChanged(new ElementChangedEventArgs<TElement>(oldElement, Element));
 
-			if (_control != null && this is IDontGetFocus)
+			if (_control != null && this is ITabStopOnDescendants)
 			{
 				_control.GotFocus += OnGotFocus;
```

Comment:
I think this part need to be protected from multiple subscription.

Like this :
```cs
_control.GotFocus -= OnGotFocus;
_control.GettingFocus -= OnGettingFocus;
_control.GotFocus += OnGotFocus;
_control.GettingFocus += OnGettingFocus;
```

Reply:
Can you or @AxelUser check whether this fixes both issues today?

Reply:
@charlesroddie @kvpt This fixes are redundant, cause control unsubsribed from events at [Dispose](https://github.com/xamarin/Xamarin.Forms/blob/2910754f338c2620d794f2eeee340fa7afb3388d/Xamarin.Forms.Platform.UAP/VisualElementRenderer.cs#L279)

Reply:
Besides it doesn't fix back the issue

Reply:
@AxelUser I'm not sure of that, from others renderers in the framework the SetElement method always follow this pattern :
```cs
if (oldElement != null)
{
     oldElement -= XXX;
}

if (newElement != null)
{
     newElement  += XXX;
}
```
And after that dispose is implement like this :
```cs
Element = null;
```

I think it's mean that the SetElement method can be called sometime several times.

Reply:
Hm, that makes sense, I've added cleanup at 034d0d6. Thank you for good point!

Reply:
Dooes this change make any difference to the issue @samhouts [identified](https://github.com/xamarin/Xamarin.Forms/pull/6468#issuecomment-508535977) ?